setup.py

```diff
@@ -134,15 +134,16 @@
     include_package_data=True,
     # This defines the interfaces to the command line scripts we"re including:
     entry_points={
-        'console_scripts': [
-            'pudl_datastore = pudl.workspace.datastore:main',
-            'pudl_setup = pudl.workspace.setup_cli:main',
-            'pudl_etl = pudl.cli:main',
-            'datapkg_to_sqlite = pudl.convert.datapkg_to_sqlite:main',
-            'ferc1_to_sqlite = pudl.convert.ferc1_to_sqlite:main',
-            'epacems_to_parquet = pudl.convert.epacems_to_parquet:main',
-            'pudl_territories = pudl.analysis.service_territory:main',
-            'datapkg_to_rst = pudl.convert.datapkg_to_rst:main',
+        "console_scripts": [
+            "censusdp1tract_to_sqlite = pudl.convert.censusdp1tract_to_sqlite:main",
+            "datapkg_to_rst = pudl.convert.datapkg_to_rst:main",
+            "datapkg_to_sqlite = pudl.convert.datapkg_to_sqlite:main",
+            "epacems_to_parquet = pudl.convert.epacems_to_parquet:main",
+            "ferc1_to_sqlite = pudl.convert.ferc1_to_sqlite:main",
+            "pudl_datastore = pudl.workspace.datastore:main",
+            "pudl_etl = pudl.cli:main",
+            "pudl_setup = pudl.workspace.setup_cli:main",
+            "pudl_territories = pudl.analysis.service_territory:main",
```

Comment:
These all changed because I alphebetized them.

Reply:
lol you would

Reply:
Less diffs later I think.